### ISSUE TITLE
how tu run visualization.py 

### ISSUE THREAD

**Issue by mariangigabyte** [scottlawsonbc/audio-reactive-led-strip#106]
i'm a noob. how to run visualization.py. what is this? it is a program?

**Comment by not-matt** [comment]
Install python 3.6, then double click visualisation.py to run it.

**Comment by mariangigabyte** [comment]
i have install anaconda. i have install python 3.6 to?

**Comment by not-matt** [comment]
You might have already installed it. Open cmd.exe, enter "python", and see
what version number is shown. If it's 3.4 or lower, install python 3.6 from
their website.

**Comment by mariangigabyte** [comment]
in start search cmd:  python is not recognized
in python file "<stdin>", line 1 in <module
name error: name python is not defined

**Comment by mariangigabyte** [comment]
C:\Users\marian\Downloads\audio-reactive-led-strip-master\python\visualisation.py .....??
i opennet with a notepad and wen i dubble click is open a notepad

**Comment by not-matt** [comment]
Generally if you're having issues that aren't explicitely related to this program, you'll get better answers by just searching them. [Here's one to get you started](https://stackoverflow.com/questions/1522564/how-do-i-run-a-python-program).

**Comment by mariangigabyte** [comment]
this visualisation.py i have to run?
C:\Users\marian\Downloads\audio-reactive-led-strip-master\python\visualisation.py

**Comment by not-matt** [comment]
Yes, that's the program you want to be running :)

**Comment by mariangigabyte** [comment]
with what program?
becouse now the default program to open is notepad and i have to make it to opening with ...python?

**Comment by not-matt** [comment]
Look at the link in my earlier answer, it explains how to do what you're
trying to do.

**Comment by mariangigabyte** [comment]
i try bat for my is star trak :)

**Comment by mariangigabyte** [comment]
if i don't open vizualization is not working the led's?

**Comment by luxxl** [comment]
python is a programming language and you can't run scripts of it without installing python. Anaconda is like more flexible easier to use python overlay so when you download anaconda you download python inside of it. You should go step by step on main page [https://github.com/scottlawsonbc/audio-reactive-led-strip](url) as it explains step by step how to get it up and running from scratch. Basicaly what you are asking is how to run it as it's already explained in main page with photos and it's super easy to follow.

**Comment by luxxl** [comment]
I believe here you tried to open python script with notepad. You can't run script with notepad, you need to run it with cmd or anaconda after when you install all of the python dependencies because otherwise it wont execute code as intended. Code by itself can be wrote on any text editor as it's basically plain text and that's why you can open it with notepad.

**Comment by mariangigabyte** [comment]
werw is the "explains step by step" 
Once everything has been configured, run visualization.py to start the visualization. The visualization will automatically use your default recording device (microphone) as the audio input.....
ehere i find the visualisation.py
the correct explain : go to Downloads\audio-reactive-led-strip-master\python and run the visualisation.py
the samething
7.In config.py:
Set N_PIXELS to the number of LEDs in your LED strip (must match......
where is the config.py
not everyone is smart like you
it is first time i instal python and anaconda and i'dont know mach
if you whant 2 felp. ok. if not. stai at home

**Comment by mariangigabyte** [comment]
and now i install python and i change a open tipe with pythoru and i 2click and nothing happens
i make steps again in my second pc and is the same
...i miss something i'm not anderstand mach inglish ant this terms...i do my best 2 make this progect

**Comment by mariangigabyte** [comment]
if you whant 2 make step by step. make a video how intall everifing

**Comment by luxxl** [comment]
go to search and open anaconda prompt and try dragging visualization.py file in it and hit enter if you done everything correctly beforehand you should be running script

**Comment by luxxl** [comment]
config.py is in same folder as visualisation.py if you don't find it then just go and download zip file of this git hub page and get all the files from python folder somewhere extracted then you should open config.py file with notepad but I recommend installing notepad ++ as it's more user friendly for editing code then find all of the stuff you need to edit like number of pixels ip address of esp.

**Comment by luxxl** [comment]
It might be hard to follow stuff for you as you are new to this kind of stuff and it's not user friendly as it's scripts that you can edit almost on the fly as it's not user executable programs that are compiled and then built for simple run

**Comment by mariangigabyte** [comment]
i dragg and enter and nothing happning

**Comment by luxxl** [comment]
weird, no errors?

**Comment by mariangigabyte** [comment]
no.wen i press enter is show again first line

**Comment by luxxl** [comment]
had u opened cmd or anaconda prompt?

**Comment by mariangigabyte** [comment]
anaconda promt

**Comment by mariangigabyte** [comment]
best way 2 help me is teamviewer :))

**Comment by luxxl** [comment]
did you run commands in prompt 
conda install numpy scipy pyqtgraph
pip install pyaudio
?

**Comment by mariangigabyte** [comment]
in anacinda promt

**Comment by luxxl** [comment]
now download whole python folder and place it somewhere on you computer then edit number of pixels as long as ip adress and drag visualization into conda prompt

**Comment by mariangigabyte** [comment]
i don't understand what U sad last time
a install notepad ++ and i set 
 UDP_IP = '192.168.1.150' like in my ws2812_controller...and
N_PIXELS = 60  
i don't see LED_PIN = 18 in my ws2812_controller....wach again

**Comment by mariangigabyte** [comment]
in confis isLED_PIN = 18
now is LED_PIN = D4 ...i changeit...os ok?

**Comment by joeybab3** [comment]
If it's getting 0fps than it's not receiving frames from the python script. Do you have the visualization window opening on your computer?

**Comment by mariangigabyte** [comment]
I ken't open a visualization.py . reed up

**Comment by not-matt** [comment]
@mariangigabyte mail me at u2237314@mvrht.net with your email address, then I'll get in touch and walk you though it over skype/discord.

**Comment by mariangigabyte** [comment]
it's working now manny 10x for not-matt

**Comment by joeybab3** [comment]
If that is true, can you close this issue?

**Comment by mariangigabyte** [comment]
How i run 2 LED strips...enibody knows?
